src/ducks/transactions/Transactions.jsx

```diff
@@ -225,7 +224,7 @@ export class TransactionsDumb extends React.Component {
 
     return (
       <>
-        <SelectionBar transactions={transactions} />
+        <SelectionBar transactions={this.transactions} />
```

Comment:
Je comprends pas comment `this.transactions` n'est pas `undefined`... ?

Reply:
Avec le componentWillMount

Reply:
ah pardon en effet !